test/core/httpRouter/httpRouter.test.js

```diff
@@ -149,12 +149,13 @@ describe('core/httpRouter', () => {
           const apiRequest = handler.firstCall.args[0];
 
           should(apiRequest).be.instanceOf(Request);
-          should(apiRequest.context.protocol).be.exactly('http');
-          should(apiRequest.context.connectionId).be.exactly('requestId');
-          should(apiRequest.input.headers).be.eql({
+          should(apiRequest.context.connection.protocol).be.exactly('http');
```

Comment:
That unit test was using deprecated properties. Those still exist, but only as links to the correct location.